### PR TITLE
chore: add usage docs

### PR DESCRIPTION
**Pull request by Slickstef11**
Added usage docs

#### Checklist

- [X] run `make test`
- [X] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [X] tests and/or benchmarks are included
- [X] documentation is changed or added
- [X] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by jensneuse — CHANGES_REQUESTED** [review]
Please remove the package-lock.json from the commit.

File: docs-website/src/pages/usage-information.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    "anonymousID": "2Ie8ynG6f1hTKs3EVjItsy0DBcn"
+  }
+}
+```
+
+## What about Sensitive Data (e.g. Secrets)?
+
+We do not collect any metrics which may contain sensitive data.
+
+We take your privacy and our security very seriously. WunderGraph telemetry falls under the security disclosure policy.
```
Comment (jensneuse) [inline docs-website/src/pages/usage-information.md:70]: "our security"?

**Review by jensneuse — CHANGES_REQUESTED** [review]

File: docs-website/src/pages/usage-information.md (hunk 1 above)
Comment (StarpTech) [reply docs-website/src/pages/usage-information.md:70]: Could you suggest something?

Comment (Pagebakers) [reply docs-website/src/pages/usage-information.md:70]: `and the security of our platform`

**Review by Pagebakers — APPROVED** [review]
LGTM after improving the security line.

File: docs-website/src/pages/usage-information.md (hunk 1 above)
Comment (StarpTech) [reply docs-website/src/pages/usage-information.md]: thx, done.

**Review by StarpTech — APPROVED** [review]
LGTM

**Review by jensneuse — APPROVED** [review]

**Pull request merged**